src/python/pants/backend/jvm/tasks/jvm_compile/compile_context.py

```diff
@@ -8,18 +8,20 @@
 import zipfile
 from contextlib import contextmanager
 
+from pants.backend.jvm.targets.jar_library import JarLibrary
 from pants.build_graph.aliased_target import AliasTarget
 from pants.build_graph.target import Target
 from pants.util.contextutil import open_zip
 
 
 def _resolve_strict_dependencies(target):
```

Comment:
It seems JarLibrary does not have "strict_deps" field. Is it intentional? If so, then does that mean JarLibrary is default to be transitive?